features/steps/ubuntu_advantage_tools.py

```diff
@@ -182,7 +182,7 @@ def when_i_create_local_ppas(context, release, next_release):
     from features.steps.machines import given_a_machine
 
     # We need Kinetic or greater to support zstd when creating the PPAs
-    given_a_machine(context, "mantic", "lxd-container", machine_name="ppa")
+    given_a_machine(context, "jammy", "lxd-container", machine_name="ppa")
```

Comment:
The comment above specifies that this needs to be Kinetic or later. Was `mantic` not working here? Does `jammy` work here?

Reply:
`mantic` fails here which resulted in the upgrade errors but `jammy` works here. I recently ran it with 'noble' and that worked too so will upgrade to noble.

Reply:
Sorry - that makes sense about mantic. I'm surprised jammy worked, but yes noble is the better option here. Thank you for updating it!